src/docs/metrics/historical-balance/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Historical Balance
+author: Yury
+date: 2022-04-21
+description: Metric returns historical balance for a chain address / contract
+---
+
+## Definition
+
+Metric returns historical balance for a chain address / contract
```

Comment:
We'll need a separate page in the /details that has a description of what `balance` is and link it here. Ideally, definition should provide more information than the one found in the name of the metric.